reliability/core-it/build.gradle.kts

```diff
@@ -13,21 +14,30 @@ import org.jetbrains.kotlin.gradle.dsl.JvmTarget
 
 plugins {
     // Build
-    id("com.android.library")
+    id("com.android.application")
```

Comment:
in this case you need to remove `androidLibraryConfig` call below, because there is no `LibraryExtension` registered.

Reply:
yup missed that